### PR TITLE
checks for $is_syncing

### PR DESCRIPTION
**Pull request by markconroy**

### CODE REVIEW

**Review by millnut — APPROVED** [review]

**Review by ekes — CHANGES_REQUESTED** [review]
This is a content item https://api.drupal.org/api/drupal/core%21modules%21menu_link_content%21src%21Entity%21MenuLinkContent.php/class/MenuLinkContent/8.9.x , so will not be being imported, so I think this should be added on config import.

File: localgov_demo.install [hunk 1]
```diff
@@ -11,40 +11,39 @@ use Drupal\menu_link_content\Entity\MenuLinkContent;
...
+      // Load block and set disabled.
+      $services_home_block = \Drupal::configFactory()->getEditable($block_id);
+      $services_home_block->set('region', 'disabled');
+      $services_home_block->save();
+    }
+
+    // Set weight of localgov_alert_banner block to float up above the banner.
+    $alert_banner_block = \Drupal::configFactory()->getEditable('block.block.localgov_alert_banner');
+    $alert_banner_block->set('weight', '-10');
+    $alert_banner_block->save();
```
Comment (andybroomfield) [inline localgov_demo.install]: Lets be careful not to re-enable this, maybe rebase from 3.x?

Comment (ekes) [reply localgov_demo.install]: I doubt it'll rebase as it's indenting all the code.

But the main code has updated. Lemme just make it again.

**Comment by ekes** [comment]
OK does someone @andybroomfield, maybe, want to give that a second pair of eyes?

**Comment by ekes** [comment]
MT @stephen-cox to double check.

**Review by andybroomfield — APPROVED** [review]
This is working for me, in terms of the rouge alert banner block config with just a weight key is no longer present in the export.

**Review by stephen-cox — APPROVED** [review]
:+1:

**Comment by markconroy** [comment]
@ekes you requested changes on this. Are you okay with how it is now?

**Comment by ekes** [comment]
I also made changes :laughing: hence waiting on a second opinion.

I'll merge.

**Pull request merged**